Perlang.Tests/Operator/Comparison.cs

```diff
@@ -0,0 +1,268 @@
+        {
+            string source = @"
+                print 1 <= 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("True", output);
+        }
+
+        [Fact]
+        public void less_than_or_equals_same_is_true()
+        {
+            string source = @"
+                print 2 <= 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("True", output);
+        }
+
+        [Fact]
+        public void less_than_or_equals_smaller_is_false()
+        {
+            string source = @"
+                print 2 <= 1;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("False", output);
+        }
+
+        //
+        // Tests for the > operator
+        //
+
+        [Fact]
+        public void greater_than_smaller_is_false()
+        {
+            string source = @"
+                print 1 > 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("False", output);
+        }
+
+        [Fact]
+        public void greater_than_same_is_false()
+        {
+            string source = @"
+                print 2 > 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("False", output);
+        }
+
+        [Fact]
+        public void greater_than_smaller_is_true()
+        {
+            string source = @"
+                print 2 > 1;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("True", output);
+        }
+
+        //
+        // Tests for the >= operator
+        //
+        [Fact]
+        public void greater_than_or_equals_smaller_is_false()
+        {
+            string source = @"
+                print 1 >= 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("False", output);
+        }
+
+        [Fact]
+        public void greater_than_or_equals_same_is_true()
+        {
+            string source = @"
+                print 2 >= 2;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("True", output);
+        }
+
+        [Fact]
+        public void greater_than_or_equals_smaller_is_true()
+        {
+            string source = @"
+                print 2 >= 1;
+            ";
+
+            string output = EvalReturningOutput(source).SingleOrDefault();
+
+            Assert.Equal("True", output);
+        }
+
+        //
+        // Special tests for 0 and -0, ensuring that positive and negative zero compares as the same value.
+        //
+        [Fact(Skip = "Fails with 'Operand must be a number' error")]
```

Comment:
I discovered when implementing these tests that this is broken. Issue filed about it: https://github.com/perlun/perlang/issues/45